### PR TITLE
set group display name; close #80

### LINKED ISSUE #80: set group display name in tests
Many of our validation tests throw an exception like this:

ERROR: class com.google.gct.idea.appengine.validation.ResourceParameterInspection: group display name should be overridden or configured via XML class com.google.gct.idea.appengine.validation.ResourceParameterInspection
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:115)
	at com.intellij.codeInspection.InspectionProfileEntry.getGroupDisplayName(InspectionProfileEntry.java:201)
	at com.intellij.codeInspection.ex.InspectionToolWrapper.getGroupDisplayName(InspectionToolWrapper.java:132)
	at com.intellij.codeInspection.ui.InspectionResultsView.addTool(InspectionResultsView.java:464)
	at com.intellij.codeInspection.ui.DefaultInspectionToolPresentation.addProblemElement(DefaultInspectionToolPresentation.java:234)

This isn't really a failure but it does hide real failures. Figure out how to fix this.

### PR DESCRIPTION
**Pull request by elharo**
@patflynn @loosebazooka

### CODE REVIEW

**Comment by elharo** [comment]
PTAL. This actually pulled out another issue in the tests. All commits are now squashed.

File: common-lib/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ def ideaHome = '../idea-IC/'
 dependencies {
     compile files('lib/GoogleFeedback.jar')
     compile fileTree(dir: ideaHome + '/lib')
-    compile "org.mockito:mockito-all:1.9.5"
+    compile files('third_party/mockito/mockito-all-1.9.5.jar')
```
Comment (patflynn) [inline common-lib/build.gradle]: isn't this in a separate PR?

Comment (elharo) [inline common-lib/build.gradle]: yes, that's already been pushed and merged into this client. I think Github is showing a diff against the state of the code whehn this branch was created.

**Comment by patflynn** [comment]
LGTM

**Pull request merged**